cypress/fixtures/opensearch_dashboards/data.json.txt

```diff
@@ -0,0 +1,4123 @@
+{
```

Comment:
do you think we can compress these large files and update the helper function to modify it? also, not sure if the ndjson vs json.txt was intentionally i think we should standardize our saved objects